androidApp/src/main/java/dev/johnoreilly/confetti/MainActivity.kt

```diff
@@ -64,28 +81,50 @@ class MainActivity : ComponentActivity() {
                         navController = navController,
                         windowSizeClass = windowSizeClass,
                         displayFeatures = displayFeatures,
+                        confettiRepository = confettiRepository,
+                        defaultConferenceParameter = defaultConferenceParameter,
                     )
                 }
             }
         }
     }
+
+    /**
+     * This relies on our deep link looking either like
+     * `confetti://confetti/sessions/{conference}`
```

Comment:
if we want to do this, we should probably pull up our navigation code like SessionDestination to shared code, and handle it the same across wear and mobile.

I can follow up with this.